.flake8

```diff
@@ -1,6 +1,18 @@
 [flake8]
 max-line-length=100
 application_import_names=snekbox
-ignore=P102,B311,W503,E226,S311
-exclude=__pycache__, venv, .venv, tests
+ignore=
+    P102,B311,W503,E226,S311,
+    # Missing Docstrings
+    D100,D104,D107,
+    # Docstring Whitespace
+    D202,D203,D204,D212,D214,D215,
+    # Docstring Quotes
+    D301,D302,
+    # Docstring Content
+    D401,D402,D405,D406,D407,D408,D409,D410,D411,D412,D413,D414
```

Comment:
The ignores haven't been finalised yet. `D400` has been ignored recently as well https://github.com/python-discord/seasonalbot/pull/146#issuecomment-474549550.

Reply:
Add D400 to ignores